packages/runtime/test/src/lib/txn.ts

```diff
@@ -0,0 +1,184 @@
+import { types } from "@algo-builder/web";
+import { stringToBytes } from "@algo-builder/web/build/lib/parsing";
+import { Account } from "algosdk";
+import { assert } from "chai";
+import { encodeBase64 } from "tweetnacl-ts";
+
+import { AccountStore } from "../../../src";
+import { convertToBuffer } from "../../../src/lib/parsing";
+import { encTxToExecParams } from "../../../src/lib/txn";
+import { Runtime } from "../../../src/runtime";
+import { AccountStoreI, EncTx } from "../../../src/types";
+import { useFixture } from "../../helpers/integration";
+
+describe("Convert encode Tx to ExecParams", function () {
+  let john: AccountStoreI;
+  let smith: AccountStoreI;
+
+  let runtime: Runtime;
+  let execParams: types.ExecParams;
+  this.beforeEach(() => {
+    john = new AccountStore(1e9);
+    smith = new AccountStore(1e9);
+
+    runtime = new Runtime([john, smith]);
+  });
+
+  // helper - help convert and check param from EncTx to ExecParams
+  function assertConvertParams (runtime: Runtime, execParams: types.ExecParams): void {
+    const [encTx] = runtime.createTxnContext(execParams);
+    const sign = {
+      sign: types.SignType.SecretKey,
+      fromAccount: execParams.fromAccount
+    };
+    // add approvalProgram and clearProgram to encTx
+    if (execParams.type === types.TransactionType.DeployApp) {
+      encTx.approvalProgram = execParams.approvalProgram;
+      encTx.clearProgram = execParams.clearProgram;
+    }
```

Comment:
I think i saw this code previously. Can we reuse in a function?

Reply:
I think those function only use for test. No idea how to reuse it.